.github/actionconfig.go

```diff
@@ -0,0 +1,116 @@
+package actionconfig
```

Comment:
@chase-crumbaugh Basically went with this structure because go's `embed` doesn't allow you to specify parent directories (no `.` or `..`). So to access the source of the generation yml, I can only be in root or `.github` or `.github/workflows`. Last option is a special directory and first one specifies `package main` (can't have same directory specify multiple pkgs in Go). Hence, went with `.github`.

Reply:
No strong opinion here, seems like you thought it through 👍